acceptance/lib/helper.rb

```diff
@@ -61,7 +61,9 @@ def self.get_option_value(value, legal_values, description,
 
     # precedence is environment variable, option file, default value
     value = ((env_var_name && ENV[env_var_name]) || value || default_value)
-    if value
+    if value == ""
+      value = default_value
```

Comment:
Seems like this function makes an assumption that `value` will always come from an options file and that, in all of the cases where the option is set, the `value` needs to be converted to a symbol?  Wasn't clear to me that this is a safe assumption to make in all cases, in which case doing the conversion closer to the caller might make more sense.

I'll leave it up to @waynr as to what's best to do here.  Just seemed a bit confusing / fragile to me.  But I'll admit that I don't have a very good grasp of what's going on here.